.github/workflows/mrc_gh_pages.yml

```diff
@@ -0,0 +1,29 @@
+name: Build and Publish MRC Docs
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_TOOLCHAIN: "nightly-2021-11-07"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout 🛎️
+      uses: actions/checkout@v3
+    - name: Install latest nightly
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: ${{ env.RUST_TOOLCHAIN }}
+        default: true
+    - name: Setup Cache 📦
+      uses: Swatinem/rust-cache@v1
+    - name: Build MRC related documentation 📝
+      run: ./scripts/mrc_docs.sh
+    - name: Deploy MRC docs to gh-pages 🚀
+      uses: JamesIves/github-pages-deploy-action@4.0.0
+      with:
+        branch: gh-pages
+        folder: ./target/doc
```

Comment:
Standard workflow, mrc docs will be published to gh-pages branch and can be served just like our spec documents